google/resource-snippets/pubsub-v1/alternatives/tf/pubsub.tf

```diff
@@ -0,0 +1,64 @@
+provider "google" {
+  project = "{PROJECT_ID}"
+  region  = "{REGION}"
+  zone    = "{ZONE}"
+}
+
+variable "deployment" {
+ default={}
+
+}
+
+variable "project_id" {
+ default={}
+
+}
+
+resource "google_pubsub_topic" "{TOPIC_REF}" {
+  name = "{TOPIC_NAME}"
+  labels= {
+    goog-dm=var.deployment
+  }
+}
+
+resource "google_pubsub_topic" "{BACKUP_TOPIC_REF}" {
+  name="{BACKUP_TOPIC_NAME}"
+  labels= {
+    goog-dm=var.deployment
+  }
+}
+
+resource "google_pubsub_subscription" "{SUBSCRIPTION_REF}" {
```

Comment:
I substituted all placeholders except "{SUBSCRIPTION_NAME}"and "{TOPIC_NAME}". ```terraform fmt``` does not have an issue formatting these because they're wrapped by quotes. However I can still fill in values if needed

Reply:
And should I add static values in the DM config yaml as well if needed?